wiki/UI-component/Video.md

```diff
@@ -39,6 +39,147 @@ The [[Video]] component utilizes the embeddable Kaltura player <sup>[[r2](#resou
 
```

Comment:
I think we need to add [Image](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/wiki/Image) as a dependency here.